src/main/java/teammates/common/util/Const.java

```diff
@@ -1384,6 +1387,11 @@ private Const() {
         public static final String FEEDBACK_SESSION_REMINDERSSENT =
                 "Reminder e-mails have been sent out to those students and instructors. "
                 + "Please allow up to 1 hour for all the notification emails to be sent out.";
+        public static final String FEEDBACK_SESSION_ACCESS_LINKS_RESENT =
+                "Recent feedback session access links have been sent out to the student who requested for a resent. "
```

Comment:
... have been sent ~~out~~ to the student who requested for a ~~resent~~ resend.